indonesia/style.css

```diff
@@ -0,0 +1,11 @@
+.rect {
+  height: 30vmin;
```

Comment:
I think there was a .flagContainer class that was already there that would have been jim dandy for this.